### PR TITLE
List-terms command.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: cmd/list_terms.go [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+	"github.com/juju/gnuflag"
+	"github.com/juju/persistent-cookiejar"
+	"gopkg.in/macaroon-bakery.v1/httpbakery"
+
+	"github.com/juju/terms-client/api"
+)
+
+const (
+	listTermsDoc = `
+list-terms show a list of Terms and Conditions documents owned by you
```
Comment (cmars) [inline cmd/list_terms.go]: List "Terms and Conditions" documents owned by you ...

File: cmd/list_terms.go [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+or one of the public groups you belong to.
+Examples
+list-terms --groups test-group1,test-group2
```
Comment (cmars) [inline cmd/list_terms.go]: How are we going to reconcile this command with the existing `charm terms` command?

File: cmd/list_terms.go [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+}
+
+var newIDMClient = func(idmURL string, client *httpbakery.Client) IDMClient {
+	return &idmClient{
+		idmURL: idmURL,
+		client: client,
+	}
+}
+
+type IDMClient interface {
```
Comment (cmars) [inline cmd/list_terms.go]: godoc please

File: cmd/list_terms.go (hunk 1 above)
Comment (alesstimec) [inline cmd/list_terms.go]: done

File: cmd/list_terms.go (hunk 2 above)
Comment (alesstimec) [inline cmd/list_terms.go]: don't know.. suggestions?

File: cmd/list_terms.go (hunk 3 above)
Comment (alesstimec) [inline cmd/list_terms.go]: done

File: cmd/list_terms.go (hunk 2 above)
Comment (cmars) [inline cmd/list_terms.go]: I think we discussed this... we'll just replace it, as this command implements the currently documented behavior.

**Comment by cmars** [comment]
:shipit:

**Comment by cmars** [comment]
:shipit:

**Comment by cmars** [comment]
:shipit:

**Pull request merged**